Packs/Trello/Integrations/Trello/action_types.csv

```diff
@@ -0,0 +1,67 @@
+,acceptEnterpriseJoinRequest
```

Comment:
What is the purpose of this file?

Reply:
This file can be removed from the PR if preferred. It contains teh list of every possible card action, which we use to generate the update-card options.

Reply:
ok so please lets either remove it from the PR or move it to a directory named `test_data` inside the `Packs/Trello/Integrations/Trello` directory